### DISCUSSION TITLE
keyframe cut accuracy - previous segment includes following segment's frame

### DISCUSSION THREAD

**Discussion by junsukha** [mifi/lossless-cut#2256] [Q&A]
Hi,

I'm using "create segments from keyframes" feature to get segments of videos.
I see there's "shift all start times" feature in Export. Is there a feature to shift all end times also?

My problem is that, after export, most of the segments' last frame is from the next segments. 
For example, if segment1 was 1-10th frame and segment2 was 11-20, after exporting the segments, segment1 includes the first frame of segment2, i.e., 11th frame. In this case, keyframe is 11th frame.

**Comment by mifi** [comment]
Hi. I will implement it
